petstore/clients/javascript/test/sample.spec.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { PetStoreClient } from "@unbranded/petstore";
+import { assert, beforeEach, it, describe } from "vitest";
+
+describe("Test Samples", () => {
+
+    let client: PetStoreClient;
+    beforeEach(async function () {
+    client = new PetStoreClient("http://localhost:5118",{
+        allowInsecureConnection: true,
+      });
+
+    });
+
+    it("pet list", async function () {
+        // list pets, return a list of pets
+        const petList = await client.pets.list();
+        assert.equal(petList.value[0].name, 'Kiwi');
+        assert.equal(petList.value[0].age, 5);
+        assert.equal(petList.value[0].ownerId, 5);
+        assert.equal(petList.value[1].name, 'Coco');
+        assert.equal(petList.value[1].age, 6);
+        assert.equal(petList.value[1].ownerId, 6);
+    });
+
+    it("pet get", async function () {
+        // get a pet, return a pet
+        const result = await client.pets.get(1);
+        assert.equal(result.name, 'Kiwi');
+        assert.equal(result.age, 5);
+        assert.equal(result.ownerId, 5);
+
+        // get a invalid pet, return error message is Invalid petId
+        try{
+            await client.pets.get(-1);
+        } catch (error) {
+            assert.equal(error.code, 0);
```

Comment:
really, this code is 0?

Reply:
pls share the link here for code and message readme

Reply:
yes. https://github.com/allenjzhang/typespec-e2e-demo/blob/main/petstore/servers/aspnet/implemented_apis.md